projects/jinja/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+RUN git clone https://github.com/pallets/jinja
+WORKDIR jinja
+
+COPY build.sh $SRC/
```

Comment:
Can you please add a [dictionnary]( https://github.com/google/fuzzing/blob/master/dictionaries/jinja2.dict ) since the format is super-sparse? Likely the jinj2 one as well at the html one.